docs/bibliography/diffusion-mri.md

```diff
@@ -1,18 +1,32 @@
 # Resources for learning the basics of diffusion MRI
 
-## Here as some basic readings to introduuce the field
+## Here are some readings to introduuce the field
 
-Wandell BA (2016) Clarifying Human White Matter. Annu Rev Neurosci 39(1):annurev-neuro-070815-013815.
+**A great overview of the field:**
 
-## Automated Fiber Quantification is a method for analyzing white matter tracts
+    Wandell BA (2016) Clarifying Human White Matter. Annu Rev Neurosci 39(1):annurev-neuro-070815-013815.
```

Comment:
When these are indented, the look like  code blocks  (see here: https://deploy-preview-28--afqdocs.netlify.app/bibliography/diffusion-mri/):

Maybe use block-quotes for these instead?
```suggestion
> Wandell BA (2016) Clarifying Human White Matter. Annu Rev Neurosci 39(1):annurev-neuro-070815-013815.
```